frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -40,143 +45,200 @@ interface IProps {
-        if (isScheduled) {
-            status = (
-                <>
-                    {GreenCircle} {TranslateText('Already scheduled')}
-                </>
-            )
-        } else {
-            if (!mission.lastRun || !mission.lastRun.endTime) {
-                if (inspection.missionDefinition.inspectionFrequency) {
-                    status = (
-                        <>
-                            {RedCircle} {TranslateText('Not yet performed')}
-                        </>
-                    )
-                } else {
-                    status = TranslateText('No planned inspection')
-                }
-                lastCompleted = TranslateText('Never')
-            } else {
-                status = inspection.missionDefinition.inspectionFrequency
-                    ? getInspectionStatus(inspection.deadline!)
-                    : TranslateText('No planned inspection')
-                lastCompleted = formatDateString(mission.lastRun.endTime!)
-            }
-        }
+    return (
+        <Table.Row key={mission.id}>
+            <Table.Cell>{status}</Table.Cell>
+            <Table.Cell>
+                <Typography
+                    link
+                    onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
+                >
+                    {mission.name}
+                </Typography>
+            </Table.Cell>
+            <Table.Cell>{mission.comment}</Table.Cell>
+            <Table.Cell>{mission.area.areaName}</Table.Cell>
+            <Table.Cell>{lastCompleted}</Table.Cell>
+            <Table.Cell>{inspection.deadline ? inspection.deadline.toDateString() : ''}</Table.Cell>
+            <Table.Cell>
+                <StyledIcon
+                    color={`${tokens.colors.interactive.focus.hex}`}
+                    name={Icons.AddOutlined}
+                    size={16}
+                    title={TranslateTextWithContext('Add to queue')}
+                    onClick={() => {
+                        openDialog()
+                        setScheduledMissions([mission])
+                    }}
+                />
+            </Table.Cell>
+        </Table.Row>
+    )
+}
+
+const columns = ['Status', 'Name', 'Description', 'Area', 'Last completed', 'Deadline', 'Add to queue']
 
-        return (
-            <Table.Row key={mission.id}>
-                <Table.Cell>{status}</Table.Cell>
-                <Table.Cell>
-                    <Typography
-                        link
-                        onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
-                    >
-                        {mission.name}
+export function InspectionTable({ deck, inspections, openDialog, setSelectedMissions, ongoingMissions }: IProps) {
+    const { TranslateText } = useLanguageContext()
+
+    let navigate = useNavigate()
+
+    let cellValues = inspections
```

Comment:
```suggestion
    const cellValues = inspections
```
If its not planned to change it

Reply:
They do change because the status on the rows of the column is changing when you schedule a mission/it is ongoing